tests/cpp/test_mma.cpp

```diff
@@ -902,91 +902,6 @@ TEST_P(HopperRS, SingleTileWithTMALoadStore) {
   EXPECT_TRUE(at::allclose(cg_outputs[0], tref, 1e-5, 1e-5));
 }
 
-TEST_P(HopperRS, SingleTileWithTMALoadOuterDimNotSplit) {
```

Comment:
Now, all the tests have "OuterDimNotSplit", so there is no need to have a separate `OuterDimNotSplit` test.